config/config-default/migrate_plus.migration.local_forms_file.yml

```diff
@@ -0,0 +1,52 @@
+uuid: 7cc5d6f4-54cb-4697-bc96-fdf9449d4bf5
+langcode: en
+status: true
+dependencies: {  }
+id: local_forms_file
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags:
+  - local_forms
+migration_group: jcc
+label: 'Local Forms: Import to file'
+source:
+  plugin: csv
+  path: modules/custom/jcc_tc_migration/csv/slo-local-forms.csv
+  delimiter: ','
+  enclosure: '"'
+  ids:
+    - id
+  constants:
+    DRUPAL_FILE_DIRECTORY: 'public://'
```

Comment:
Do you want all these imported files in the base of the files directory? Or does it make any sense to put them in a named subdirectory?